codecov-cli/scripts/build_linux.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
```

Comment:
pinning 3.9 is the important detail here. If we build with newer python version, it will not work on older platforms.

Reply:
can you add this note into the code?